src/main/java/Duke.java

```diff
@@ -1,10 +1,117 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
     public static void main(String[] args) {
+        boolean isExit = false;
+        Scanner scanner = new Scanner(System.in);
+        TaskList taskList = new TaskList();
+
         String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        //Greet
+        System.out.println("---------------------------------------------------");
+        System.out.println("Duke: Hello! What can I do for you today?");
+
+        
+        while (isExit == false){
+            //Getting Input
+            System.out.println("---------------------------------------------------");
+            System.out.print("You: ");
+            String rawInput = scanner.nextLine();
+            String[] inputList = rawInput.split(" ");
+            String cmd = inputList[0];
+            String description;
+
+            //Responding
+            switch (cmd) {
+                case ("list"):
```

Comment:
Avoid magic strings, so maybe for each of these cases, you can say for ex: LIST = "list" and reference the constant in the if statement